slam_toolbox/launch/has_node_params.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2021 PAL Robotics S.L.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
```

Comment:
We have a dep on `nav2_map_server`, you can make a dep on `nav2_common` and add this to that in foxy + use it from there.